README.md

```diff
@@ -7,13 +7,34 @@ Plugin that sets up a Python environment for building and running tests using
 Usage
 -----
 
-Apply the plugin to your project and configure the associated extension:
+Apply the plugin to your project:
```

Comment:
Would it be fine to have two steps:
1. Install it according to this page
2. Use this code to add it to your project